plugins/adobe-analytics-configuration/index.ts

```diff
@@ -31,13 +36,21 @@ import { ValidationPluginResult } from '../../types/validationPlugin';
     return rsids && server;
   });
 
-  const message = valid
-    ? 'Valid! Adobe Analytics Extension has been configured!'
-    : 'Missing required configuration for Adobe Analytics';
-  const errors = valid ? [] : configurationEvents.map((event) => event.uuid);
-  return {
-    events: errors,
-    message,
-    result: valid ? 'matched' : 'not matched'
-  };
+  return valid
+  ? {
+      message: 'Valid! Adobe Analytics Extension has been configured!',
+      events: [],
+      result: 'matched'
+    }
+  : !valid && versionEvents.some(sharedStateVersions.getExtensionsKey('"com.adobe.module.analytics"'))
+  ? {
+      message: 'Missing required configuration for Adobe Analytics.',
+      events: [],
+      result: 'not matched'
+    }
+    :{
+      message: 'no Analytics extension is detected, nothing to validate.',
+      events: [],
+      result: 'matched'
```

Comment:
I think we should use `unknown` since we don't know the intent of the user and the extension could be missing.